### ISSUE TITLE
Changing log level to Info

### ISSUE THREAD

**Issue by prary** [GoogleContainerTools/skaffold#3225]
Following warning gives the impression of docker daemon necessarily install on the machine and for most of the guys docker daemon is the real daemon. Can we change log level to info instead of warning.

https://github.com/GoogleContainerTools/skaffold/blob/4dd6b59e9a2201f3f19dbe4c8ca2b616621349fa/pkg/skaffold/build/cache/retrieve.go#L128

https://github.com/GoogleContainerTools/skaffold/blob/4dd6b59e9a2201f3f19dbe4c8ca2b616621349fa/pkg/skaffold/build/cache/retrieve.go#L133

Output of the warning is bit scary too
```
time="2019-10-21T08:14:42-04:00" level=warning msg="error adding artifacts to cache; caching may not work as expected: inspecting image: error during connect: Get [http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/images/..../:bdb5677@sha256:cc7a48e/json]http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/images/..../:b5677@sha256:cc7a48e9b/json: open //./pipe/docker_engine: 
The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running."
```

**Comment by tejal29** [comment]
I think the reason why this is warn is because warning is the default log level. That ways, users always see this warning message.

**Comment by prary** [comment]
Hi @tejal29 I think I didn't phrase it correctly let me rephrase it.
Can we change 
```logrus.Warnf("error adding artifacts to cache; caching may not work as expected: %v", err) ```
to 
```logrus.Infof("error adding artifacts to cache; caching may not work as expected: %v", err) ```

warning looks too harsh and creates a impresssion as if docker needs to be installed on the machine which isn't the case :)

**Comment by prary** [comment]
Every time I run skaffold build it gives scary warning instead of happy success message which looks like a failure.

```
←[36mINFO←[0m[0026] No files changed in this command, skipping snapshotting.
←[36mINFO←[0m[0026] CMD [ "python", "app.py" ]
←[36mINFO←[0m[0026] No files changed in this command, skipping snapshotting.
time="2019-11-20T16:28:19+05:30" level=info msg="Build complete in 6m40.451s"
time="2019-11-20T16:28:19+05:30" level=info msg="Test complete in 0s"
time="2019-11-20T16:28:19+05:30" level=debug msg="FIXME: Got an status-code for which error does not match any expected type!!!: -1" module=api status_code=-1
time="2019-11-20T16:28:19+05:30" level=warning msg="error adding artifacts to cache; caching may not work as expected: inspecting image: error during connect: This error may indicate that the
aemon is not running.: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/images/pythonserver:dirty@sha256:718a0ab1cf1a1f6896605a1de97123749
82e98c68eeb66d4329d5bc1/json: open //./pipe/docker_engine: The system cannot find the file specified."
```
